web/src/components/tasks/business-formation/DbaFormationPaginator.tsx

```diff
@@ -81,6 +82,7 @@ export const DbaFormationPaginator = (): ReactElement => {
       stepperState[1].isComplete = stepIndex === 2;
       return stepperState;
     });
+    updateQueue.queueFormationData({ lastVisitedPageIndex: stepIndex }).update();
```

Comment:
we should have a test for this within the DBA flow

Reply:
Since DBA / Nexus always starts on the first step, do we even need/want to set this?

Reply:
Or should we add a note to update this once we do the upcoming Dakota story that *will* remember Nexus name availability?